test/testdata/container_storage_sleep.json

```diff
@@ -0,0 +1,49 @@
+{
+	"metadata": {
+		"name": "podsandbox-sleep"
+	},
+	"image": {
+		"image": "quay.io/crio/redis:alpine",
+		"annotations": {
+			"io.kubernetes.cri-o.Storage": "devicemapper"
+		}
+	},
+        "annotations": {
+                "io.kubernetes.cri-o.Storage": "devicemapper"
+        },
```

Comment:
nit; indenting here is off

Reply:
second nit: in server/sandbox_run_linux.go, we processed the annotation for the Storage. however, here we're adding the annotation on the container level. I would think we'd need it on the sandbox level

Reply:
third nit (sorry for not compressing into one comment and instead spamming :sweat_smile: ): there was a push to pair down the number of json files here and instead use jq directly in the bats suite to change the information in the default files. What motivation do you have for wanting this example that wouldn't be solved by jq in the bats suite